includes/updates.php

```diff
@@ -135,3 +135,26 @@ function pmpro_updates_notice_complete() {
 </div>
 <?php
 }
+

```

Comment:
Cool for now. I might move this into a separate file when we bundle all of the 2.0 updates stuff together.